src/tokenized-line.coffee

```diff
@@ -118,20 +118,27 @@ class TokenizedLine
     oddIndentLevel = @indentLevel - Math.floor(@indentLevel)
     Math.round(@tabLength * oddIndentLevel)
 
-  buildSoftWrapIndentationTokens: (token) ->
+  buildSoftWrapIndentationTokens: (token, hangingIndentationSpaces) ->
```

Comment:
@nathansobo: if I understand your question correctly, we need to break the "hanging spaces" into tokens so that we could show indent guides.

After a bit of sleep (I love Hammock Driven Development :sleeping:), however, I realized this wasn't doing what I expected: indeed indent guides weren't shown, as only one token was created for all the hanging indentation spaces.

f0b9bb7ce3e75f716c0d0297bbcacee5fe289f81 solves this by calculating the `hangingIndentLevel` and summing it to `TokenizedLine#indentLevel`.